amp/repository/aim/view/viewActivityDifferences.jsp

```diff
@@ -287,6 +339,11 @@ if(document.getElementById('method').value == "enableMerge") {
     document.getElementById('saveButton').disabled = "disabled";
     document.getElementById('saveButton').style.display = 'none';
     $('#backButton').prop('value', '<digi:trn>Back to Audit Logger</digi:trn>');
+}else if (document.aimCompareActivityVersionsForm.method.value === "compareAll"){
+    document.getElementById('saveButton').disabled = "disabled";
+    document.getElementById('saveButton').style.display = 'none';
+    $('#MyTabs').replaceWith('<font color="red" size="2%"><strong><digi:trn>&nbsp;&nbsp;List of Activities Compared to their Previous Versions</digi:trn></strong></font>');

```

Comment:
I wouldn't add this with JavaScript lets add it to the jsp code so its clear to be read

Reply:
also as I added in my first comment, it would be easier and quicker to add the split in the table (an empty tr) would do